docs/UserGuide.md

```diff
@@ -76,12 +76,17 @@ Format: `help`
 
 Adds a new entry to their address book for financial advisors, including personal details such as name, address, occupation, phone number and email.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL o/OCCUPATION a/ADDRESS [t/TAG]…​`
+Precise command format:
```

Comment:
might want to standardize this as we discussed this week?

Reply:
Rectified! Thanks for the input